### PR TITLE
Fixed PropTypes warnings

### PR DESCRIPTION
**Pull request by ansh-k**
closes https://jiraeyeem.atlassian.net/browse/ED-57

### CODE REVIEW

File: src/components/tabs/CategoryTabBar.js [hunk 1]
```diff
@@ -372,7 +373,7 @@ export const CategoryTabBar = ({
     </nav>
     <div className={categoryTabBarToolsStyle}>
       <Link
-        activeClassName="isActive"
+        activeclassname="isActive"
```
Comment (usmanchaudhryme) [inline src/components/tabs/CategoryTabBar.js:376]: Why change this? Does it have some sort of impact?

File: src/components/users/UserRenderables.js [hunk 2]
```diff
@@ -312,7 +312,7 @@ export class UserProfile extends PureComponent {
   }
   static propTypes = {
     avatar: PropTypes.object.isRequired,
-    externalLinksList: PropTypes.object,
+    externalLinksList: PropTypes.array,
```
Comment (usmanchaudhryme) [inline src/components/users/UserRenderables.js:315]: I believe it's from the upgrade of react-router, is it being passed as an array now?

File: src/components/users/UserRenderables.js [hunk 3]
```diff
@@ -340,11 +340,11 @@ export class UserProfile extends PureComponent {
   }
 
   static defaultProps = {
-    externalLinksList: null,
+    externalLinksList: () => null,
```
Comment (usmanchaudhryme) [inline src/components/users/UserRenderables.js:343]: Why a noop sort of method, why not just an empty array?

**Review by usmanchaudhryme — APPROVED** [review]
Mostly looking good, just 3 questions, you may merge accordingly.

File: src/components/users/UserRenderables.js (hunk 2 above)
Comment (ansh-k) [reply src/components/users/UserRenderables.js:315]: Yes it return as array

File: src/components/tabs/CategoryTabBar.js (hunk 1 above)
Comment (ansh-k) [reply src/components/tabs/CategoryTabBar.js:376]: I receive a warning activeClassName this should need to be in camelcase.

File: src/components/users/UserRenderables.js (hunk 3 above)
Comment (ansh-k) [reply src/components/users/UserRenderables.js:343]: I receive some warning if use empty array.

**Pull request merged**